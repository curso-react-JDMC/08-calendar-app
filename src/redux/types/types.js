export const types ={
    //ui
    uiOpenModal:'[ui] Open modal',
    uiCloseModal:'[ui] Close modal',

    //calendar
    calendarEventAddNew:'[cal] Add new',
    calendarEventSetActive:'[cal] Set active',
    calendarEventClearActive:'[cal] Clear active',
    calendarEventUpdate:'[cal] Updated',
    calendarEventDelete:'[cal] Delete'
}