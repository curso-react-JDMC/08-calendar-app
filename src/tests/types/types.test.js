import { types } from "../../redux/types/types"


describe('Pruebas en types', () => { 
    test('los types deben ser iguales', () => { 
        expect(types).toMatchSnapshot()
    })
})