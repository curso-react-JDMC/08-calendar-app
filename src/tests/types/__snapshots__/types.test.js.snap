// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en types los types deben ser iguales 1`] = `
Object {
  "authChekingFinish": "[auth] Checking finish login state",
  "authLogin": "[auth] Login",
  "authLogout": "[auth] Logout",
  "authStartLogin": "[auth] Start login",
  "authStartRegister": "[auth] Start register",
  "authStartTokenRenew": "[auth] Start token renew",
  "calendarEventAddNew": "[cal] Add new",
  "calendarEventClearActive": "[cal] Clear active",
  "calendarEventDelete": "[cal] Event Delete",
  "calendarEventLoaded": "[cal] Event Loaded",
  "calendarEventLogout": "[cal] Event Logout",
  "calendarEventSetActive": "[cal] Set active",
  "calendarEventUpdate": "[cal] Event Updated",
  "calendarStartAddNew": "[cal] Start add new",
  "uiCloseModal": "[ui] Close modal",
  "uiOpenModal": "[ui] Open modal",
}
`;
